src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -703,13 +694,11 @@ export default {
 			}
 		},
 
-		highlightAnimation() {
-			// trigger CSS highlight animation by setting a class
-			this.$refs.message.classList.add('highlight-animation')
-		},
-		highlightAnimationStop() {
-			// when the animation ended, remove the class, so we can trigger it again another time
-			this.$refs.message.classList.remove('highlight-animation')
+		highlightMessage() {
+			this.isHighlighted = true
+			setTimeout(() => {
+				this.isHighlighted = false
+			}, 5000)
```

Comment:
PS: we can also use `animationend` event in Vue same as other events with `@animationend`

Reply:
But I'm fine with 5s too

Reply:
`animationend` is actually better, as we specify animation length only in styles, not in JS code